services/frontend/src/scenes/Login/Login.js

```diff
@@ -2,11 +2,32 @@ import React from "react";
 import connect from "./connect";
 import LoginLayout from "./LoginLayout";
 
-const Login = ({errorMessage, isEmailInvalid, isLoading, onLogin}) => (
+const errorMessageMap = (error) => {
```

Comment:
I had a misunderstanding with where we try to put all of the business logic between our `connect`, `.js` and `layout.js` files. 

That said from your comment further down,
`the more business logic that can be pushed to the connect level ... the better. `
I can see why it would be at the top level so I've moved it back to the connect file.

As for changing it from an error message to object, that is just a result of changing `createRequestSlice` to be able to get more error information.